lib/travis/build/templates/header.sh

```diff
@@ -85,6 +85,22 @@ function travis_nanoseconds() {
   $cmd -u $format
 }
 
+travis_internal_ruby() {
+  if ! type rvm &>/dev/null; then
+    source ~/.rvm/scripts/rvm &>/dev/null
+  fi
+
+  local internal_ruby
+  internal_ruby="$(
+    rvm list \
+      | awk -F- '/ruby-(2\.[0-2]\.[0-9]|1\.9\.3)/ { print $2 }' \
```

Comment:
@BanzaiMan I'm sure a solution is possible to select a future 2.1.10 over 2.1.5, but I'm also certain the code will be more convoluted, and I don't expect we'll have to really care about such a scenario.  WDYT?